solutions/instances/variables.tf

```diff
@@ -43,6 +43,151 @@ variable "prefix" {
+  type        = string
+  description = "The name of the IBM Cloud Logs instance to create. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
+  default     = "cloud-logs"
+}
+
+variable "cloud_logs_region" {
+  description = "The IBM Cloud region where Cloud Logs instances will be created."
+  type        = string
+  default     = "eu-es"
+}
+
+variable "cloud_logs_tags" {
+  type        = list(string)
+  description = "Tags associated with the IBM Cloud Logs instance (Optional, array of strings)."
+  default     = []
+}
+
+variable "cloud_logs_service_endpoints" {
+  description = "The type of the service endpoint that will be set for the IBM Cloud Logs instance."
+  type        = string
+  default     = "public-and-private"
+  validation {
+    condition     = contains(["public", "private", "public-and-private"], var.cloud_logs_service_endpoints)
+    error_message = "The specified service_endpoints is not a valid selection"
+  }
+}
+
+variable "enable_cloud_logs_data" {
+  type        = bool
+  description = "Enable data bucket for a IBM Cloud Logs instance"
+  default     = true
+}
+
+variable "enable_cloud_logs_metrics" {
+  type        = bool
+  description = "Enable metrics bucket for a IBM Cloud Logs instance"
+  default     = true
+}
+
+variable "enable_en_integration" {
+  type        = bool
+  description = "Enable event notification integration for IBM Cloud Logs instance"
+  default     = true
+}
+
+variable "existing_en_instance_crn" {
+  type        = string
+  description = "The CRN of existing event notification instance. If not supplied, a new instance is created."
+  default     = null
+}
+
+variable "cloud_logs_retention_period" {
+  type        = number
+  description = "The number of days IBM Cloud Logs will retain the logs data in Priority insights."
+  default     = 7
+}
+
+variable "cloud_log_data_bucket_name" {
+  type        = string
+  default     = "cloud-logs-data-bucket"
+  description = "The name of the Cloud Object Storage bucket to create to store cloud log data. Cloud Object Storage bucket names are globally unique. If the `add_bucket_name_suffix` variable is set to `true`, 4 random characters are added to this name to ensure that the name of the bucket is globally unique. If the prefix input variable is passed, the name of the instance is prefixed to the value in the `<prefix>-value` format."
+}
+
+variable "cloud_log_metric_bucket_name" {
+  type        = string
+  default     = "cloud-logs-metrics-bucket"
+  description = "The name of the Cloud Object Storage bucket to create to store cloud log data. Cloud Object Storage bucket names are globally unique. If the `add_bucket_name_suffix` variable is set to `true`, 4 random characters are added to this name to ensure that the name of the bucket is globally unique. If the prefix input variable is passed, the name of the instance is prefixed to the value in the `<prefix>-value` format."
```

Comment:
This variable is to store metric data not log data. So description needs a change.